tests/cypress/integration/filter.spec.js

```diff
@@ -0,0 +1,269 @@
+      .within(() => {
+        cy
+          .get(nsTeal('filter_panel-filter_active_vars_contents'))
+          .should('have.length.gte', 1);
+
+        cy
+          .get(nsTeal('filter_panel-ADSL_filter-filter-cards'))
+          .children()
+          .should('have.length', 0);
+
+        cy
+          .get(nsTeal('filter_panel-ADAS_filter-filter-cards'))
+          .children()
+          .should('have.length', 0);
+
+        cy
+          .get(nsTeal('filter_panel-ADTTE_filter-filter-cards'))
+          .children()
+          .should('have.length', 0);
+
+        cy
+          .get(nsTeal('filter_panel-ADLB_filter-filter-cards'))
+          .children()
+          .should('have.length', 0);
+      });
+  });
+
+  //  Add variables to filter
+  // ######################################
+
+  it('Add filter variables has shiny content rendered', {
+    defaultCommandTimeout: 10000,
+  }, () => {
+    cy
+      .get('@add_filter')
+      .within(() => {
+        cy
+          .get('.shiny-input-container')
+          .each(($el) => {
+            cy
+              .wrap($el)
+              .children()
+              .should('have.length.gte', 1);
+          });
+      });
+  });
+
+  //  Add variables to filter
+  // ######################################
+
+  it('Add filter', () => {
+    const nsInfoTable = (selector) => `${nsTeal('filter_panel-teal_filters_info-table')} ${selector}`;
+
+    // Add Age filter (find and click on it)
+    // ------------------------------------
+    cy
+      .get('@add_filter')
+      .within(() => {
+        cy
+          .get('.shiny-input-container:first')
+          .contains('Select variable to filter')
+          .should('be.visible')
+          .parent()
+          .as('filterButton')
+
+        cy.get('@filterButton').click('top');
+
+        cy.get('.dropdown-menu.open li').contains('Age').as("age")
+
+        cy.get("@age").click('top');
+      });
+
+    // Let shiny finish rendering
+    cy
+      .get('html')
+      .not('.shiny-busy');
+
+    // Test if application has non-zero subjects
+
+    cy
+      .get(nsInfoTable('table tbody tr:first td:last'))
+      .contains(/[1-9]+[0-9]*\/[1-9]+[0-9]*/);
+
+    // Monitor how many times summary is recalculated vias shiny:recalculated
+    // event
+    cy
+      .get('@filter_summary')
+      .within(() => {
+        cy
+          .get('.shiny-bound-output')
+          .invoke('on', 'shiny:recalculated', cy.stub().as('summary_change'));
+      });
+
+    // Move slider via cypress2 mouse drag
+    // ------------------------------------
+    cy
+      .get('@filter_variables')
+      .within(() => {
+        // Make sure that the overlay plot is rendered
+        cy
+          .get('.filterPlotOverlayRange .shiny-plot-output')
+          .children()
+          .should('have.length.gte', 1);
+
+        cy
+          .get('.filterPlotOverlayRange .shiny-plot-output')
+          .not('.recalculating');
+
+        // Move the handle 3x
```

Comment:
I don't like this solution, but I can't figure out a clean way of doing this.

It's not 100% deterministic, but doing it 3x helps guarantee that the slider is moved at least once.